src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    <p className={`${person.age ? `Person__age` : ''}`}>
+      {`${person.age ? `I am ${person.age}` : ``}`}
+    </p>
+
+    <p className="Person__partner">
+      {`${person.isMarried
+        ? `${person.partnerName} is my ${person.isMarried && person.sex === 'm'
+          ? `wife`
+          : `husband`}`
```

Comment:
for better readability you can create separate variable in component like `partnerStatus` and use it in jsx
```suggestion
  const partnerStatus = sex === 'm'
     ? 'wife'
     : 'husband';
     ...
     `${partnerName} is my ${partnerStatus}`
```